src/components/pages/MyCases/MyCases.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './MyCases.less';
+import { Typography, Table, Button } from 'antd';
```

Comment:
Great job using AntD and less on this new feature. This will make it easier when we get to switching all of the stylings over in the future!